packages/shared/src/utils/contract/contract.ts

```diff
@@ -32,13 +32,13 @@ export const isValidToSell = (contract_info: TContractInfo) =>
 
 export const hasContractEntered = (contract_info: TContractInfo) => !!contract_info.entry_spot;
 
-export const isAccumulatorContract = (contract_type: string) => /ACCU/i.test(contract_type);
```

Comment:
I added empty string as a default param because `contract_type` could be possibly `undefined` as we passed it as `contract_info.contract_type` inside of function and `TContractInfo` has :
`contract_type?: string;` (node_modules/@deriv/api-types/index.d.ts, line 13467)
Moreover, this approach is being used in line 51 (this file) and 143.